HIP/hip-1056.md

```diff
@@ -1490,18 +1473,22 @@ message FilteredItemHash {
 #### Block Proof
 
 The block proof carries the trust of the consensus network to the contents of the block. A block can conceptually be
-represented as a binary merkle tree with 4 sub-trees at depth 2 which create the block proof and tree when hashed to
+represented as a binary merkle tree with 8 sub-trees at depth 2 which create the block proof and tree when hashed to
 obtain a merkle root.
 
 ![Root Hashes](../assets/hip-1056/root_hashes.svg)
```

Comment:
the svg file name here and the actual one in assets don't match, `_` here v.s. `-` in the actual filename